java/pom.xml

```diff
@@ -518,18 +518,7 @@
                                     <!--Set by groovy script-->
                                     <directory>${native.cudf.path}</directory>
                                     <includes>
-                                        <include>libcudf_ast.so</include>
-                                        <include>libcudf_base.so</include>
-                                        <include>libcudf_comms.so</include>
-                                        <include>libcudf_hash.so</include>
-                                        <include>libcudf_interop.so</include>
-                                        <include>libcudf_io.so</include>
-                                        <include>libcudf_join.so</include>
-                                        <include>libcudf_merge.so</include>
-                                        <include>libcudf_partitioning.so</include>
-                                        <include>libcudf_reductions.so</include>
-                                        <include>libcudf_replace.so</include>
-                                        <include>libcudf_rolling.so</include>
```

Comment:
I don't know much about the background of this. Is cudf now going to be single library? can you point me to some more references on this so I can educate myself on the background of this?

Reply:
nm. just saw your description